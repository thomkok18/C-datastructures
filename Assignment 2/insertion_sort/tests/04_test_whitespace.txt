 51

2
3
 69
 69
5
 666
7
8
 42
700
