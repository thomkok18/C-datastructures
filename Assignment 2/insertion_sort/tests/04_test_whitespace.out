2
3
5
7
8
42
51
69
69
666
700
